### ISSUE TITLE
[18.10.1] Upgrade 18.10.0 to 18.10.1 issue

### ISSUE THREAD

**Issue by kozfelipe** [centreon/centreon#7070]
Upgrading 18.10.0 to 18.10.1 via packages generate incorrect entry at /etc/centreon-broker/watchdog.xml

`<log><![CDATA[/var/log/centreon-broker/watchdog.log/watchdog.log]]></log>`

causing cbd service to fail
```
cbwd: watchdog: log: could not open file '/var/log/centreon-broker/watchdog.log/watchdog.log': Not a directory
systemd: cbd.service: main process exited, code=exited, status=1/FAILURE
```

**Comment by kduret** [comment]
Thanks for feedback.
We are aware about this, and we are going to release Centreon Web 18.10.3 asap

**Comment by lolokai** [comment]
This bug is not corrected on Centreon Web 18.10.2

**Comment by lpinsivy** [comment]
Corrected by PR #7367
